main.css

```diff
@@ -0,0 +1,17 @@
+* {
```

Comment:
Except for `border-sizing` try to stay away from `*` selectors since the `*` selector will have to rerender the page for every single element.

Instead if you are trying to do something on a global scale (like a full page background or changing all the fonts) use the `body` or `html` selectors.

You can see this issue since every line of text starts over with a new background instead of sharing a single one:

![screen shot 2015-07-04 at 3 26 31 pm](https://cloud.githubusercontent.com/assets/2532004/8509284/2d23f73e-2261-11e5-9899-aa32e1b4ba00.png)

I've added a single px black line to show where things are being effected by a `*` rule:

![screen shot 2015-07-04 at 3 27 10 pm](https://cloud.githubusercontent.com/assets/2532004/8509286/3f80c6e6-2261-11e5-8115-7358f29c2133.png)